### ISSUE TITLE
Pagination overview breaks when there are no pages.

### ISSUE THREAD

**Issue by Mippy** [Shopkeepers/Shopkeepers#668]
* **Shopkeepers version:**  `19aee4f` (Compiled from Source)
* **Spigot version:** `This server is running CraftBukkit version git-Spigot-0287a20-7560f5f (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)`

If you have no shops, /shopkeeper list shows blank for "Page x of x".

- Start fresh with no shops.
- Do /shopkeeper list

The result will say "Page  of  ".
<img src="https://imgs-i.sabel.dev/SUnkaRN.png"/>

Should probably say "Page 0 of 0" or not show at all.

I have no interfering plugins (all the plugins on the server I made myself, or completely unrelated such as viaversion, plugman, permissionsex)

Might fix later depending how I'm feelin idk I found this while working on a PR for /shopkeeper list

**Comment by blablubbabc** [comment]
Works fine for me: http://prntscr.com/tm5v60
Have you made any changes to your config? Or maybe you updated the plugin and there have been config / message changes

**Comment by Mippy** [comment]
That is really strange. I haven't modified any code or made any config changes.

**Comment by blablubbabc** [comment]
Try deleting your config and letting the plugin freshly generate it. If this doesn't help, can you post your config?

**Comment by Mippy** [comment]
Alright. Stopped server, deleted config, rebooted server and it was still broken.


HOWEVER I became suspicious of BLC because of how chat formatting has changed with 1.16.
I loaded up vanilla 1.16 and.... it worked. ._. How embarrassing I didn't possibly think that could be a conflicting thing until I added some logger lines to see the chat message after it was processed and it being totally fine with numbers.

To be fair I probably shouldn't be developing and testing on a modded client anyway.

Sorry about that!
Tldr: BLC Bug, not plugin error

**Comment by blablubbabc** [comment]
I have no idea what BLC is, but I am glad you figured it out!
